addons/website/static/src/scss/website.scss

```diff
@@ -1278,6 +1278,95 @@ $-o-carousel-controllers-size: map-get($spacers, 5);
     }
 }
 
+// Single scroll animation for the carousel
+.o_slide_single {
+    --o_flex_value: 50%;
+    /* Set up css variables
+    (Define 1 to 4 cards in the carousel) */
+    .o_with_1_cards {
+        --o_flex_value: 100%;
+    }
+
+    .o_with_2_cards {
+        --o_flex_value: 50%;
+    }
+
+    .o_with_3_cards {
+        --o_flex_value: 33.33%;
+    }
+
+    .o_with_4_cards {
+        --o_flex_value: 25%;
+    }
```

Comment:
I used this  (`--o_flex_value`) to properly display the number of cards in the carousel. This value is also retrieved by the JS so that it knows how many cards are in the carousel and can properly handle the cycle.

**What do you think of this practice? Defining a CSS variable that is later retrieved by JS? Should I go for a data-attribute instead for the JS part?**